src/modules/translation/languages/es.json

```diff
@@ -963,6 +963,7 @@
     "open_in_editor": "Abrir en el editor"
   },
   "item": {
+    "blockchain_id": "ID en la Blockchain",
```

Comment:
Im not 100% sure. Maybe we can use blockchain id for this too. Wdyt?

Reply:
Both work for me, but I used an "almost" real representation of the phrase in Spanish thinking that it would match the language.

Reply:
a real reprensentation may be then `Id en la cadena de bloques`

Reply:
That's the correct translation, but I used the words `ID` and `Blockchain` that are fairly known and identify concepts in both languages mixed with Spanish so the phrase wasn't entirely in english, that's why the "almost real" representation.
I changed it to `Blockchain ID` though.